src/index.js

```diff
@@ -1,3 +1,24 @@
-import { inputData } from './js/inputData.js';
+import _ from 'lodash'; // eslint-disable-line
+import displayScore from './js/inputData.js';
+import { postScore, getByApi } from './js/postScores.js';
 
-inputData();
+const myApi = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/5YR6tabo7tvHObEZexNL/scores';
+const scoresForm = document.getElementById('scoresForm');
+const refreshBttn = document.getElementById('refreshBttn');
+
+refreshBttn.addEventListener('click', () => {
+  getByApi(myApi).then((response) => {
+    displayScore(response.result);
+  });
+});
+
+scoresForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  postScore();
+  scoresForm.reset();
+});
+
+getByApi(myApi).then((res) => {
+  displayScore(res.result);
+  console.log(res.result);
```

Comment:
- [ ] This `console.log` goes against the JavaScript requirement quoted below. There is another in the `addGame.js` file. Please consider doing away with all debugging statements.
```suggestion
```

> Do not commit console.log to your repo. It’s ugly, it kills performance and it can make confidential data be visible to anyone using the browser tools to look at your website.